doc.config.json

```diff
@@ -78,6 +78,10 @@
               {
                 "title": "PlanetScale and Prisma",
                 "path": "/docs/guides/serverless-api-with-planetscale-and-prisma.md"
+              },
+              {
+                "title": "Using Git Actions",
+                "path": "/docs/guides/git-actions"
```

Comment:
not sure about this, but does git-actions need .md at the end?

Reply:
Yes, updated